tests/arista_eos/show_vrf/arista_eos_show_vrf.raw

```diff
@@ -1,14 +1,14 @@
-   Vrf              RD                     Protocols       State             Interfaces  
---------- -------- --------------- ----------- ----------
-   blue             10.125.253.15:1        ipv4,ipv6       v4:routing,       Vlan1006, Vlan2230, Vlan2231, 
-                                                           v6:no routing     Vlan2232, Vlan2233, Vlan2234,
-                                                                             Vlan2235, Vlan2236, Vlan2237,
-                                                                             Vlan2238, Vlan2239, Vlan2240
-   green            <not set>              ipv4,ipv6       v4:routing,       Vlan1015, Vlan1016, Vlan1017,
-                                                           v6:no routing     Vlan1018, Vlan1019, Vlan1020,
-                                                                             Vlan1021
-   yellow           10.125.253.15:4        ipv4,ipv6       v4:routing,       Vlan1009                      
-                                                           v6:no routing                                
-   red              10.125.253.15:6        ipv4,ipv6       v4:routing,                               
-                                                           v6:no routing                                  
-   black            999:999                ipv4,ipv6       v4:no routing     Management1
+  Vrf          RD                   Protocols      State             Interfaces            
```

Comment:
Do you mind reverting this as well? I'd rather just keep the raw data we already have since I'm not sure if this is something that has changed between versions or not.

Reply:
Spacing is critical, without proper spacing in the table, none of this works.  The original raw data had spacing all over the place and was missing the critical \s$ at the end of each line for some of the lines.  The original template was also not concerned with collecting interfaces.  It appears as though the original raw file was input incorrectly as every version of eos we have tested does not care about the terminal width it always puts out a table exactly the same width with two interfaces max per line with proper spacing.  I only accounted for three interfaces per line in my update because that is what is in the existing raw file

Reply:
Ok that's fair and hopefully doesn't become an issue in the future or we'll have to address those differences in spacing.

Reply:
Murphy hates me so it will probably be me.